pkg/oc/admin/migrate/authorization/authorization.go

```diff
@@ -1,285 +0,0 @@
-package authorization
-
-import (
-	"errors"
-	"fmt"
-	"io"
-
-	"k8s.io/api/rbac/v1beta1"
-	kerrs "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/client-go/util/flowcontrol"
-	rbacinternalversion "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/rbac/internalversion"
-	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
-	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
-	"k8s.io/kubernetes/pkg/kubectl/resource"
-
-	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
-	"github.com/openshift/origin/pkg/authorization/registry/util"
-	"github.com/openshift/origin/pkg/oc/admin/migrate"
-	"github.com/openshift/origin/pkg/oc/cli/util/clientcmd"
-
-	"github.com/spf13/cobra"
-)
-
-var (
-	internalMigrateAuthorizationLong = templates.LongDesc(`
-		Check for parity between Openshift authorization objects and Kubernetes RBAC
-
-		A controller is used to keep Openshift authorization objects and Kubernetes RBAC in sync.
-		This command checks for parity between those objects across all namespaces and reports
-		all objects that are out of sync.  These objects require manual intervention to sync
-		as the controller handles all cases where automatic sync is possible.
-
-		The following resource types are checked by this command:
-
-		* clusterrole
-		* role
-		* clusterrolebinding
-		* rolebinding
-
-		No resources are mutated.`)
-
-	// errOutOfSync is retriable since it could be caused by the controller lagging behind
-	errOutOfSync = migrate.ErrRetriable{errors.New("is not in sync with RBAC")}
-)
-
-type MigrateAuthorizationOptions struct {
-	migrate.ResourceOptions
-	rbac rbacinternalversion.RbacInterface
-}
-
-func NewCmdMigrateAuthorization(name, fullName string, f *clientcmd.Factory, in io.Reader, out, errout io.Writer) *cobra.Command {
```

Comment:
With this gone I believe `ResourceOptions.Complete` can be updated:

https://github.com/openshift/origin/blob/cd29885133474a180ef77ce596a9675f584920a1/pkg/oc/admin/migrate/migrator.go#L101-L108

Reply:
> With this gone I believe ResourceOptions.Complete can be updated:

ok